tests/Fingerprint.tests.js

```
@@ -311,6 +328,13 @@ QUnit.test( 'setAliases()', function( assert ) {
 		fingerprint.hasAliases( 'en', enAliases ),
 		'Set aliases passing a MultiTermMap object.'
 	);
+
+	fingerprint.setAliases( emptyAliases );
```

This setter is used wrong and should throw an exception.